### PR TITLE
CE-1623 Adding clicktracking for items on special insights page

### PR DESCRIPTION
**Pull request by kamilkoterba**
https://wikia-inc.atlassian.net/browse/CE-1623

Added clicktracking for items on Special:Insights apge

@Wikia/community-engineering

### CODE REVIEW

**Comment by lgarczewski** [comment]
LGTM, merge when ready.

File: extensions/wikia/Insights/scripts/LoopTracking.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+		var insightType;
+
+		function onLinkClick() {
+			// Track a click on a notification link
+			var trackingParams = {
+				trackingMethod: 'both',
+				category: 'insights',
+				action: tracker.ACTIONS.CLICK_LINK_TEXT,
+				label: insightType+'insight-list-item'
```
Comment (adamkarminski) [inline extensions/wikia/Insights/scripts/LoopTracking.js]: I think we miss a dash '-' before the word insight. Can you also change the word 'insight' to 'insight*s*' to match the css classes etc.?

File: extensions/wikia/Insights/templates/Insights_subpageList.php [hunk 2]
```diff
@@ -17,7 +17,7 @@
 			<p class="insights-header-description"><?= wfMessage( InsightsHelper::INSIGHT_DESCRIPTION_MSG_PREFIX . $subpage )->escaped() ?></p>
 		</div>
 		<div class="insights-content">
-			<ul class="insights-list">
+			<ul class="insights-list" data-type="<?= $subpage ?>">
```
Comment (adamkarminski) [inline extensions/wikia/Insights/templates/Insights_subpageList.php]: Sanitize the output

File: extensions/wikia/Insights/scripts/LoopTracking.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			};
+			tracker.track(trackingParams);
+		}
+
+		function init() {
+			var $insightsList = $('.insights-list'),
+				$insightsListItems;
+			insightType = $insightsList.data('type');
+			$insightsListItems = $insightsList.find('.insights-list-item-title');
```
Comment (adamkarminski) [inline extensions/wikia/Insights/scripts/LoopTracking.js]: Do you need to save it to a variable? You can chain .find() and .each().

**Pull request merged**